tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -44,339 +37,19 @@
-                ["Astrophysics", 0.9941025972366333],
-                ["Phenomenology-HEP", 0.0034253709018230438],
-                ["Instrumentation", 0.0025460966862738132],
-                ["Gravitation and Cosmology", 0.0017545684240758419],
-            ],
-            "status_code": 200
-        }
-    elif data.get('corpus') == "keywords":
-        return {
-            "labels": [
-                ["galaxy", 0.29424679279327393],
-                ["numerical calculations", 0.22625420987606049],
-                [
-                    "numerical calculations: interpretation of experiments",
-                    0.031719371676445007
-                ],
-                ["luminosity", 0.028066780418157578],
-                ["experimental results", 0.027784878388047218],
-                ["talk", 0.023392116650938988],
-            ],
-            "status_code": 200
-        }
-
-
-def fake_refextract_extract_references_from_file(*args, **kwargs):
-    """Mock refextract extract_references_from_file func."""
-    return {'references': []}
-
-
-def get_halted_workflow(app, record, extra_config=None):
-    extra_config = extra_config or {}
-    with mock.patch.dict(app.config, extra_config):
-        workflow_uuid = start('article', [record])
-
-    eng = WorkflowEngine.from_uuid(workflow_uuid)
-    obj = eng.processed_objects[0]
-
-    assert obj.status == ObjectStatus.HALTED
-    assert obj.data_type == "hep"
-
-    # Files should have been attached (tarball + pdf, and plots)
-    assert obj.files["1407.7587.pdf"]
-    assert obj.files["1407.7587.tar.gz"]
-
-    assert len(obj.files) > 2
-
-    # A publication note should have been extracted
-    pub_info = obj.data.get('publication_info')
-    assert pub_info
-    assert pub_info[0]
-    assert pub_info[0].get('year') == 2014
-    assert pub_info[0].get('journal_title') == "J. Math. Phys."
-
-    # A prediction should have been made
-    prediction = obj.extra_data.get("relevance_prediction")
-    assert prediction
-    assert prediction['decision'] == 'Non-CORE'
-    assert prediction['scores']['Non-CORE'] == 0.8358207729691823
-
-    expected_experiment_prediction = {
-        'experiments': [
-            {'label': 'CMS', 'score': 0.75495152473449707}
-        ]
-    }
-    experiments_prediction = obj.extra_data.get("experiments_prediction")
-    assert experiments_prediction == expected_experiment_prediction
-
-    keywords_prediction = obj.extra_data.get("keywords_prediction")
-    assert keywords_prediction
-    assert {
-        "label": "galaxy",
-        "score": 0.29424679279327393,
-        "accept": True
-    } in keywords_prediction['keywords']
-
-    # This record should not have been touched yet
-    assert "approved" not in obj.extra_data
-
-    return workflow_uuid, eng, obj
+from mocks import (
+    fake_download_file,
+    fake_beard_api_request,
+    fake_magpie_api_request,
+    fake_beard_api_block_request,
+    fake_refextract_extract_references_from_file,
+)
```

Comment:
Are you sure that this is going to work? How you can import things here if there are not inside a module